src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,48 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+repositories=\
+  eclipse/microprofile-samples,\
```

Comment:
I came across [this](https://fusebit.io/blog/github-api-list-repositories/?utm_source=www.google.com&utm_medium=referral&utm_campaign=none) article, which demonstrates how to retrieve all repos belonging to an org. Could we use that to list the repos instead of storing a hard copy? I think you can list the repos for the org of eclipse and then filter out the repos not starting with `microprofile-`.

Reply:
Thank you @Emily-Jiang for the article. 
Currently, this feature is out of the scope of this PR, but  the microprofile- repo feature is already part of the backlog and traced via https://github.com/microprofile/microprofile-site/issues/104.